lib/openstudio-standards/standards/necb/ECMS/nv.rb

```diff
@@ -0,0 +1,187 @@
+        # puts "outdoor_air_flow_per_floor_area us #{outdoor_air_flow_per_floor_area}"
+
+        ##### Get heating/cooling setpoint temperature schedules from the osm file
+        # These schedules are used for min/max Tin schedules under the objects of "ZoneVentilation:DesignFlowRate" and "ZoneVentilation:WindandStackOpenArea".
+        # Note: as per E+ I/O Ref.: "If the user enters a valid schedule name, the minimum/maximum temperature values specified in this schedule will override the constant value specified in the Minimum/Maximum Indoor Temperature field." under the objects of "ZoneVentilation:DesignFlowRate" and "ZoneVentilation:WindandStackOpenArea".
+        if thermal_zone.thermostat.is_initialized
+          if thermal_zone.thermostat.get.to_ThermostatSetpointDualSetpoint.is_initialized
+            if thermal_zone.thermostat.get.to_ThermostatSetpointDualSetpoint.get.heatingSetpointTemperatureSchedule.is_initialized ||
+                thermal_zone.thermostat.get.to_ThermostatSetpointDualSetpoint.get.coolingSetpointTemperatureSchedule.is_initialized
+              zone_thermostat = thermal_zone.thermostatSetpointDualSetpoint.get
+              zone_clg_thermostat_sch = zone_thermostat.coolingSetpointTemperatureSchedule.get
+              zone_htg_thermostat_sch = zone_thermostat.heatingSetpointTemperatureSchedule.get
+
+              ##### Create schedule for max Tin to have NV on the basis of cooling setpoint temperature for default day
+              zone_clg_thermostat_sch_name = zone_clg_thermostat_sch.name
+              zone_clg_sp_schedule = zone_clg_thermostat_sch.to_ScheduleRuleset.get
+              puts zone_clg_sp_schedule.name
+
+              max_Tin_schedule = zone_clg_sp_schedule.clone(model).to_ScheduleRuleset.get
+              max_Tin_schedule.setName('natural_ventilation_max_Tin_schedule')
+              ## default days/weekdays
+              max_Tin_schedule_defaultDay = max_Tin_schedule.defaultDaySchedule
+              max_Tin_schedule_defaultDay.setName('natural_ventilation_max_Tin_schedule_defaultDay')
+              max_Tin_schedule_defaultDay_times = max_Tin_schedule_defaultDay.times
+              max_Tin_schedule_defaultDay_values = max_Tin_schedule_defaultDay.values
+              max_Tin_schedule_defaultDay_values_adjusted = max_Tin_schedule_defaultDay_values.map { |i| i + setpoint_adjustment_for_nv }
+              i = 0.0
+              max_Tin_schedule_defaultDay_times.each do |time|
+                max_Tin_schedule_defaultDay.addValue(time, max_Tin_schedule_defaultDay_values_adjusted[i])
+                i += 1.0
+              end
+
+              ##### Create schedule for min Tin to have NV on the basis of cooling setpoint temperature for default day
+              zone_htg_thermostat_sch_name = zone_htg_thermostat_sch.name
+              zone_htg_sp_schedule = zone_htg_thermostat_sch.to_ScheduleRuleset.get
+              puts zone_htg_sp_schedule.name
```

Comment:
same...remove the puts unless it tell the user something.

Reply:
Removed.